Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -120,6 +121,11 @@ def __init__(self,
         # splitting Gradient
         self.split = split
 
+        # warm-start
+        self.warmstart  = warmstart
```

Comment:
Add an attribute to determine whether `warmstart` is off or on. If it's on, then define a supplementary attribute `hasstarted` to know whether proximal computations have already been done or not.